MahApps.Metro/Controls/NumericUpDown.cs

```diff
@@ -70,7 +70,13 @@ private static void IsReadOnlyPropertyChangedCallback(DependencyObject dependenc
             typeof(NumericUpDown),
             new FrameworkPropertyMetadata(default(double?), FrameworkPropertyMetadataOptions.BindsTwoWayByDefault, OnValueChanged, CoerceValue));
 
-        public static readonly DependencyProperty MinimumProperty = DependencyProperty.Register(
+      public static readonly DependencyProperty ButtonsPlacementProperty = DependencyProperty.Register(
+           "ButtonsPlacement",
+           typeof(Dock),
```

Comment:
Can you create an enumeration for this, instead using `Dock`?

something like

```csharp
enum ButtonAlignment
{
    Left,
    Right
}
```

i think all other values make no sense